.github/workflows/release-idl.yml

```diff
@@ -0,0 +1,52 @@
+# Runs whenever a push to the default branch updates the package.json file
+# of the @webref/idl package. The job publishes a new @webref/idl package to
+# npm and creates a new @webref/idl release on GitHub.
+#
+# The job will not do anything if the version number in package.json does not
+# differ from the one published on npm.
+
+name: Create and publish new @webref/idl release
+
+on:
+  push:
+    paths:
+      - 'packages/idl/package.json'  
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+
+    - name: Checkout webref
+      uses: actions/checkout@v2
+
+    - name: Prepare new release
+      run: |
+        npm ci
+        npm run prepare
+
+    - name: Test new release
+      run: npm test
+
+    - name: Publish npm package
+      id: publish
+      uses: JS-DevTools/npm-publish@v1
+      with:
+        token: ${{ secrets.NPM_TOKEN }}
+        package: ./packages/idl/package.json
+        access: public
+
+    - name: Create GitHub release
```

Comment:
I didn't create a GitHub release for any of the earliest releases, what happens if there is none, or if this last step sometimes fails?

Reply:
I'm confused by the Releases page on GitHub:
https://github.com/w3c/webref/releases

I'm the one who created a release for the CSS package. The page also lists IDL package releases, but perhaps they are just tags created with `git tag`? If you did not create the tags yourself, then Lerna most likely did.

I'll push an update to the PR to simply create a tag for now. It seems good practice to tag the commit associated with a particular npm package. We don't need something more fancy for now.

Reply:
Done in last commit. If these steps fail, then that means the npm package will not be directly associated with a commit. We should be notified of the job's failure and can always add the tag manually afterwards.